### ISSUE TITLE
Neuer Dokumenttyp Patent

### ISSUE THREAD

**Issue by alw-bsz** [OPUS4/application#1130]
Das Patent ist ein Dokumenttyp aus dem Gemeinsamen Vokabular (sogar schon aus der Fassung von 2010). Da es eine Nachfrage danach gibt und er auf Kundenanforderung in einigen Instanzen implementiert wurde, sollte er auch Teil des Kanons an Standard-Dokumenttypen in OPUS 4 werden.

Für das Patent ist die Abbildung von Metadaten in OPUS 4 erforderlich, für die bislang keine Felder vorhanden sind. Die Tabelle document_patents und die darin vorhandenen Felder können nicht verwendet werden, weil sie für Patent-Informationen bei anderen Dokumenttypen gedacht sind. Konkret werden zusätzliche Felder für diese Angaben benötigt:

| Standardisiertes Feld | Standardisierte Bezeichnung des Enrichments beim BSZ | Anmerkung |
| -- | -- | -- |
| Inhaber des Patents / Anmelder der Offenlegung | PatentOwner |  |
| Nummer des Patents / der Offenlegung | PatentNumber |  |
| Datum der Patentanmeldung / Offenlegung | PatentDateApplied | Ob Datum oder Jahr ist ggf. nicht ganz so wichtig. Es sollten aber keine weiteren "Dopplungen" wie PatentYearApplied und PatentDateApplied implementiert werden. |
| Patentamt | PatentOffice | |
| Datum der Patenterteilung | PatentDateGranted | |

Eine Offenlegung (genauer: Offenlegungsschrift) ist die Erstveröffentlichung einer Patentanmeldung. Sie ist die Vorstufe zur geprüften Patentschrift.

Evtl. macht es Sinn, die Angabe "Datum der Patentanmeldung / Offenlegung" in zwei Felder "Datum der Offenlegung" und Datum der Patentanmeldung" zu trennen.

Die Angaben zu Patenten als eigenem Dokumenttyp sollten auch im UI klar von den Patentinformationen als weitere Angaben bei anderen Dokumenttypen getrennt werden (vgl. auch https://github.com/OPUS4/application/issues/617).

**Comment by j3nsch** [comment]
Für Standard-OPUS 4 müssten die Enrichments ein Prefix bekommen. Dadurch, dass wir noch das alte Publish-Modul verwenden, müssen die Enrichments auch vorab angelegt werden. Das ist sonst eigentlich nicht mehr notwendig. 

Es sind sicherlich auch Sonderbehandlungen in verschiedenen XSLT Dateien notwendig, oder? Es wäre vermutlich gut einen PR mit den Änderungen am BSZ für diesen Dokumenttyp zu haben, der dann die Grundlage für weitere Arbeiten bildet.

**Comment by alw-bsz** [comment]
Ja, das wirkt sich auch auf einige XSLTs aus. Ich bringe dann den Doktyp Patent mit allen Anpassungen in einem eigenen PR ein.
